### ISSUE TITLE
(bug) Weak or no LTE signal can cause unresponsive System Ui or phone (P6pro)(android14)

### ISSUE THREAD

**Issue by robovoice1** [GrapheneOS/os-issue-tracker#2640]
Yesterday i had 2 issues with many apps open and relying on internet access and very weak or lost LTE signal.

1. 
- Vanadium and Brave browsers with 1 tab open, each
- Element app open and connected
- Poweramp pro open (but not on replay)
- Vlc player replaying music
- GOS Apps app got new updates  (auto download disabled)
Then the LTE signal dropped and i got this message:
![1000060383_20231101153931](https://github.com/GrapheneOS/os-issue-tracker/assets/92784377/015f53b0-35db-4f91-9cb4-92d4d5f7a973)

After 2 minutes waiting and moving to an area with better connectivity the phone responded again.
After this i rebooted the phone.
The next issue was more disturbing.

2.
- I watched a video with newpipe
- 2 browsers open
- Element app open and connected
- poweramp app open but not on replay

Because of the doorbell ringing i put the phone in screenlock mode. When i came back the whole phone was unresponsive:

- Fingerprint sensor, no reaction
- Swiping up to enter pw not possibble
- pressing the power button showed the restart, emergengy.... buttons, but tapping on them with no reaction 
 I could see in the top right that the LTE signal dropped again (very low)
After moving to an area with better connecrivity and waiting for 5 minutes the phone responded again.

So basically i was locked out of my own phone for about 10 minutes  because of very low or no LTE signal.
The odd thing was , no app crash, no system ui message, nothing like this was displayed before or after this. 🤯

It is known that in this area here are white spots for LTE/5G. 

Pixel 6 pro raven
UP1A.231005.007.2023103100
Vanadium 119.0.6045.66

**Comment by thestinger** [comment]
It's likely not caused by what you think it is. We have existing issues open about Android 14 regressions causing lag. Do you use multiple user profiles?

**Comment by robovoice1** [comment]
no only owner profile active

**Comment by robovoice1** [comment]
oh and i forgot to mention: netguard with invizible pro via socks 5 also connected and running in both situations.

.................and i could not see any stingray devices intercepting my connection here ( a joke, a joke).

**Comment by robovoice1** [comment]
To be more serious:
It is (for me) always the same pattern,  very low LTE/5G signal and the "fun" begins on the smartphone.

**Comment by thestinger** [comment]
Does not seem related.

**Comment by robovoice1** [comment]
when this hapoen again, i send logs.  (With android 13 i never had this issue with the same phone)

**Comment by thestinger** [comment]
Seems fixed.
